Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Christopher Loggins</authors>
+        <owners>Christopher Loggins</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.0" />
```

Comment:
I thought adding >=2.0, <3.0 assumes that FIE will never make the mistake of introducing a breaking change into 2.x.